tests/acceptance/features/apiGraph/getGroup.feature

```diff
@@ -105,3 +105,18 @@ Feature: get groups and their members
     When user "Brian" gets all the members information of group "tea-lover" using the Graph API
     Then the HTTP status code should be "401"
     And the last response should be an unauthorized response
+
+
+  Scenario: Get details of a group
+    Given group "tea-lover" has been created
+    When user "Alice" gets details of a group "tea-lover" using the Graph API
+    Then the HTTP status code should be "200"
+    And the json responded should contain a group with these key and value pairs:
+      | key         | value      |
+      | displayName | tea-lover  |
+      | id          | %group_id% |
+
+
+  Scenario: admin user tries to get group information of non-existing group
+    When user "Alice" gets details of a group "nonExistingGroup" using the Graph API
```

Comment:
@PrajwolAmatya now we create different steps to handle not existing user/group . More detail present in this  issues https://github.com/owncloud/ocis/issues/5646 and discussed PR https://github.com/owncloud/ocis/pull/5609#discussion_r1116493927